pkg/util/util.go

```diff
@@ -125,6 +125,22 @@ func GetPodOrdinals(tc *v1alpha1.TidbCluster, memberType v1alpha1.MemberType) (s
 	return podOrdinals, nil
 }
 
+func GetDeleteSlotsNumber(annotations map[string]string) (int32, error) {
+	if len(annotations) == 0 {
+		return 0, nil
+	}
```

Comment:
```suggestion
```
I think these lines are unnecessary, because if len is 0, then in L132 `ok` will always be false.

Reply:
That's right